README.md

```diff
@@ -241,9 +240,9 @@ CTest will only display the standard error output of tests that failed. In order
 
 ## Running A Subset Of The Tests
 
-`${PROJECT_BINARY_DIR}\tests\llvm-lit\llvm-lit.py` can be invoked on a subdirectory of a testsuite and will execute all
-the tests under that subdirectory. This can mean executing the entirety of a single testsuite, running all tests under
-a category in libcxx, or running a single test in `std` and `tr1`.
+`${PROJECT_BINARY_DIR}\tests\utils\stl-lit\stl-lit.py` can be invoked on a subdirectory of a testsuite and will execute
+all the tests under that subdirectory. This can mean executing the entirety of a single testsuite, running all tests
+under a category in libcxx, or running a single test in `std` and `tr1`.

```

Comment:
Have you double-checked running a single test lately? I recall attempts failing for me today with UNRESOLVED status.

Reply:
`python tests\utils\stl-lit\stl-lit.py ..\..\llvm-project\libcxx\test\std\numerics\bit\bit.endian\endian.pass
.cpp`, for example:
```
-- Testing: 1 tests, 1 workers --
UNRESOLVED: libc++ :: std/numerics/bit/bit.endian/endian.pass.cpp (1 of 1)
********************
Unresolved Tests (1):
  libc++ :: std/numerics/bit/bit.endian/endian.pass.cpp


Testing Time: 0.26s
  Unresolved Tests: 1
```

Reply:
Ah, my reading comprehension is poor: "Can be invoked on a subdirectory", which sometimes invokes a single test. Nevermind.

Reply:
Ran into the same thing. So thanks for answering a question I didn't ask :D.

Reply:
So lit is supposed to be able to run a single test but the way they do it is incompatible with the fact that a single test file can produce several tests to be run for us. It would require an upstream change to enable pointing at a single file.